### ISSUE TITLE
'closed' field in Sprint incorrectly mapped

### ISSUE THREAD

**Issue by wheweldc** [rcarz/jira-client#64]
Looking at src/main/java/net/rcarz/jiraclient/greenhopper/Sprint.java#deserialise

I think this line:
        closed = Field.getBoolean(map.get("closed"));

should be something like:
        closed = "CLOSED".equalsIgnoreCase(Field.getString(map.get("state")));

I would have been happy to push a bug branch for a pull request, but it's not allowed. What is the procedure for supplying fixes to GitHub projects?

BTW. Very useful project, cheers.

**Comment by rcarz** [comment]
Thanks. Unfortunately the GreenHopper API is inconsistent from release to release and not well supported by Atlassian. I think the only way to really support GH is to add checks and handle multiple API versions. So rather than change that line, a check for the existence of "closed" in the map would probably be best.

GitHub generally requires you to fork projects, push changes to your fork, and then submit a pull request.

**Comment by wheweldc** [comment]
As in:
    closed = map.containsValue("CLOSED");
??

Yuk. it works, but yuk. :-}

Would you like me to fork and send a pull request?

**Comment by rcarz** [comment]
Yes it's awful but unfortunately they keep changing the API. A pull request would be appreciated... otherwise I'll find the time to do it when I can.
